tests/src/performance/Scenario/JitBench/Runner/Benchmark.cs

```diff
@@ -39,6 +40,14 @@ public virtual Metric[] GetDefaultDisplayMetrics()
             return new Metric[] { Metric.ElapsedTimeMilliseconds };
         }
 
+        /// <summary>
+        /// Does this benchmark run properly on a given architecture?
+        /// </summary>
+        public virtual bool IsArchitectureSupported(Architecture arch)
+        {
+            return (arch == Architecture.X86 || arch == Architecture.X64);
```

Comment:
Maybe this could just return true so that it wouldn't block tests on arm/arm64 in the future